src/foam/nanos/crunch/Capability.js

```diff
@@ -53,7 +55,12 @@ foam.CLASS({
     {
       name: 'description',
       class: 'String',
-      documentation: `Description of capability`
+      documentation: `Brief description of capability`
+    },
+    {
+      name: 'longDescription',
```

Comment:
This strikes me as unnecessary. Can you give an example of when we'd want both a short and long description? Are they both user-facing?

Reply:
My thinking is `description` would be a summary like "Make USD Payments", while `longDescription` would be a block of text explaining the capability

Reply:
This seems reasonable to me. Like an entry in an app store which has both a one line and much longer description.